packages/tabs/src/Tabs/Tabs.styles.ts

```diff
@@ -34,6 +34,7 @@ export const tabContainerStyle = css`
 export const inlineChildrenWrapperStyle = css`
   display: flex;
   align-items: center;
+  align-self: flex-end;
```

Comment:
<img width="399" alt="Screenshot 2023-10-30 at 4 05 59 PM" src="https://github.com/mongodb/leafygreen-ui/assets/5740159/0d1bd3b1-a13e-4d6a-a77d-f800327458d4">

It doesn't look like the icons and button line up with the tabs to the right. Another solution could be to wrap this container in another div and inside that div add 
```
display: flex;
border: 1px .....;
```
and then remove `align-self: flex-end` from this element.

Reply:
You were right, but the fix was actually somewhere else haha
<img width="619" alt="image" src="https://github.com/mongodb/leafygreen-ui/assets/17347644/f29b3f21-03e7-43a3-9b5a-cac7195cf18f">